app/src/main/java/com/github/k409/fitflow/FitFlowApplication.kt

```diff
@@ -59,20 +61,26 @@ class FitFlowApplication : Application(), Configuration.Provider, ImageLoaderFac
 
     override fun newImageLoader(): ImageLoader {
         return ImageLoader.Builder(this)
+            // .respectCacheHeaders(false) // forces caching
+            .crossfade(true)
```

Comment:
From my testing, I believe that respectCacheHeaders(false) should be un-commented here, because without it caching did not work on my device